src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
It's not related to PR but a common recommendation.
It's better to use plural words with `-s` at the end as a constant name for any array. So you will always know that you have to work with array and you can use singular names inside array methods.
Smth like this:
```
const persons = [...]

persons.filter(person => {
  // do smth
})
```